wpeverest/everest-forms
# Tweak - Drag and drop sortable-ui design enhancement on sidebar panel

### All Submissions:






### Changes proposed in this Pull Request:



This PR enhanced the drag and drop sortable UI design for multiple choices and checkbox field option in the sidebar panel. This PR also includes validation message error on the address field and checkbox field on front-end.

### How to test the changes in this Pull Request:

1. Drag and drop checkbox field and multiple choices field on the builder.
2. Now check the choices option for General field checkbox and multiple choices.
3. Likewise, check items option for payment fields checkbox and multiple choices.
4. While dragging the choices option or items, there should be a placeholder for the dragged list with a light bordered background.

#### Regarding the validation message error
1. Make the address field and checkbox field required.
2. Try to submit the form without filling it.
3. There should not be a red mark on the optional field of the address field.
4. Now fill the form.
5. After filling the form check the red invalid input still exist or not before submitting the form.
 

### Types of changes:




### Other information:




### Changelog entry

> Fix - Validation message issue on the address field and checkbox field.
> Tweak - Drag and drop sortable-ui design enhancement on sidebar panel.
